DeepSpeech.py

```diff
@@ -3,229 +3,347 @@
 from __future__ import absolute_import
 # -*- coding: utf-8 -*-
 
+import os
+import sys
+
+log_level_index = sys.argv.index('--log_level') + 1 if '--log_level' in sys.argv else 0
+if log_level_index > 0 and log_level_index < len(sys.argv):
+    os.environ['TF_CPP_MIN_LOG_LEVEL'] = sys.argv[log_level_index]
+
 import datetime
 import json
 import numpy as np
-import os
 import shutil
 import subprocess
-import sys
 import tempfile
 import tensorflow as tf
 import time
+import importlib
+import cgi
+import pickle
 
 from collections import OrderedDict
-from math import ceil
+from math import ceil, floor
 from tensorflow.contrib.session_bundle import exporter
 from tensorflow.python.tools import freeze_graph
```

Comment:
I re-checked with 1.0 branch and in fact they fixed freeze_graph, it does not perform flag parsing on import anymore, but rather under "__main__" name, so when being called as a CLI tool.